src/api/archive_unstable_storageDiff.md

```diff
@@ -28,25 +28,41 @@
     - `hash`: The result contains the hexadecimal-encoded hash of the storage entry.
   - `childTrieKey` (optional): String containing the hexadecimal-encoded key of the child trie of the "default" namespace. If this field is not present, the storage difference is calculated for the main storage trie.
 
-**Return value**: A JSON object.
+**Return value**: String containing an opaque value representing the operation.
+
+## Notifications format
+
+This function will later generate one or more notifications in the following format:
+
+```json
+{
+    "jsonrpc": "2.0",
+    "method": "archive_unstable_storageDiffEvent",
+    "params": {
+        "subscription": "...",
+        "result": ...
+    }
+}
+```
+
+Where `subscription` is the value returned by this function, and `result` can be one of:
+
+### storageDiff
```

Comment:
Perhaps just remove this line, since this object is no longer returned by this function?

Reply:
I've removed the `The JSON object returned by this function has the following format`. Dq: is that the line mentioned? 🤔